cloud_experts_tutorials/cloud-experts-dynamic-certificate-custom-domain.adoc

```diff
@@ -509,19 +524,20 @@ set-cookie: 52e4465485b6fb4f8a1b1bed128d0f3b=68676068bb32d24f0f558f094ed8e4d7; p
 cache-control: private
 ----
 
+[id="cloud-experts-dynamic-certificate-custom-domain-debug"]
 == Debugging
 [NOTE]
 ====
 The validation process usually takes 2-3 minutes to complete while creating certificates.
 ====
 
-If you face issues during the certificate create step, run `oc describe` against each of the `certificate`,`certificaterequest`,`order`, and `challenge` resources to view the events or reasons that will help identify the cause of the issue.
+If you face issues during the certificate create step, run `oc describe` against each of the `certificate`,`certificaterequest`,`order`, and `challenge` resources to view the events or reasons that can help identify the cause of the issue.
```

Comment:
I'm not clear on which step this troubleshooting is supposed to apply to. Certificate creation makes it sound like this is to troubleshoot "Create a certificate resource to provision a certificate for the custom domain Ingress Controller" under "Creating a custom domain Ingress Controller", but I think it's actually referring to "Annotate the route to trigger cert-manager to provision a certificate for the custom domain:" under "Deploy a sample application".

It might be worth adding this as a Troubleshooting dot-heading under "Deploying a sample application", or changing the title to be specific: "Troubleshooting dynamic certificate provisioning" as the heading, and "If annotating your route does not trigger certificate creation" or similar

Reply:
Thanks-this helps to make it clearer.